packages/twenty-front/src/modules/settings/integrations/utils/AdminAccounts.ts

```diff
@@ -1,4 +1,4 @@
-export const ADMIN_ACCOUNTS = ['jordan@insuros.ca', 'ronnie@insuros.ca'];
+const ALLOWED_ADMIN_DOMAIN = 'insuros.ca';
```

Comment:
```suggestion
const ALLOWED_ADMIN_DOMAIN = '@insuros.ca';
```

without this someone could enter `bob+insuros.ca@gmail.com`.

You could instead rewrite the below part to be:
```ts
const [localPart, domain] = email.split('@');
  return domain === ALLOWED_ADMIN_DOMAIN && !localPart.toLowerCase().includes('demo');
  ```

Reply:
safe

Reply:
i'll have to leave out the `@` in the `ALLOWED_ADMIN_DOMAIN` tho, splitting around `@` breaks around the @ symbol. That guarantees the domain is after the `@`